aria-practices.html

```diff
@@ -53,23 +53,39 @@
 
   <section id="intro">
     <h2>Introduction</h2>
-    <p>This section is <em>informative.</em></p>
     <p>
-      WAI-ARIA Authoring Practices is a guide for understanding how to use
-      <cite><a href="https://www.w3.org/TR/wai-aria-1.2/">WAI-ARIA 1.2</a></cite> to create an accessible Rich Internet Application.
-      It provides guidance on the appropriate application of WAI-ARIA, describes recommended WAI-ARIA usage patterns, and explains concepts behind them.
+      The ARIA Authoring Practices Guide (APG) explains how to make rich internet applications accessible to users of assistive technologies and keyboard interfaces by developing web sites that properly convey accessibility semantics and implement common keyboard conventions.
+      It also provides guidance on some closely related topics, such as supporting operating system settings for high contrast and reduced motion.
     </p>
     <p>
-      Languages used to create rich and dynamic web sites, e.g., HTML, JavaScript, CSS, and SVG, do not natively include all the features required to make sites usable by people who use assistive technologies (AT) or who rely on keyboard navigation.
+      Accessibility semantics are the meaning of user interface elements that need to be conveyed to assistive technology users in order for those users to understand and use the elements.
+      For example, people who can visually perceive a search icon button understand the element can be activated to perform a search by the way it is styled and positioned.
+      To make that icon accessible to a screen reader user, one of the semantics that needs to be communicated is that the element represents a button, and there are several ways that semantic can be conveyed to assistive technologies in code.
+      In addition, for the button to be accessible via a keyboard, it needs to be keyboard focusable, and it is commonly expected that when focused, pressing <kbd>Enter</kbd> or <kbd>Space</kbd> will activate it.
```

Comment:
Change "for the button to be accessible via a keyboard, it needs to be keyboard focusable," to "for the button to be accessible via keyboard it needs to be keyboard focusable," (removed comma after keyboard and "a" before "keyboard")

Reply:
I simplified the sentence to be:

>In addition, keyboard accessibility requires the button to be focusable, and keyboard conventions call for pressing <kbd>Enter</kbd> or <kbd>Space</kbd> to activate the button when it is focused.